webview_panels/src/modules/documentationEditor/components/documentationPropagation/DocumentationPropagation.tsx

```
@@ -100,45 +254,44 @@ export const DocumentationPropagationButton = ({
         iCurrColumns = [];
         break;
       }
+      const tempColumnLoadingState: Record<string, boolean> =
+        currColumns.reduce(
+          (acc, curr) => ({ ...acc, [curr.column]: false }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)